docs/src/test-advanced-js.md

```diff
@@ -368,15 +368,15 @@ const { chromium } = require('@playwright/test');
 module.exports = async config => {
   const { baseURL, storageState } = config.projects[0].use;
   const browser = await chromium.launch();
-  const context = await browser.newPage();
+  const context = await browser.newContext();
   const page = await context.newPage();
   try {
     await context.tracing.start({ screenshots: true, snapshots: true });
     await page.goto(baseURL);
     await page.fill('input[name="user"]', 'user');
     await page.fill('input[name="password"]', 'password');
     await page.click('text=Sign in');
-    await context.storageState({ path: storageState });
+    await context.storageState({ path: 'storageState' });
```

Comment:
Oh I see, I had only changed it because my IDE was complaining about it:
![Screenshot_20220616_194047](https://user-images.githubusercontent.com/98180468/174195526-a0872040-6d9a-4c68-80fc-2684ae4237d4.png)

Then I'll revert that commit.

Reply:
See https://github.com/microsoft/playwright/pull/14932#discussion_r899644630